components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -216,17 +216,17 @@
 
     private String sessionId;
 
-    private OpenIDConnectClaimFilterImpl openIDConnectClaimFilter;
+    private static OpenIDConnectClaimFilterImpl openIDConnectClaimFilter;
     private static final Log diagnosticLog = LogFactory.getLog("diagnostics");
 
-    public OpenIDConnectClaimFilterImpl getOpenIDConnectClaimFilter() {
+    public static OpenIDConnectClaimFilterImpl getOpenIDConnectClaimFilter() {
 
         return openIDConnectClaimFilter;
     }
 
-    public void setOpenIDConnectClaimFilter(OpenIDConnectClaimFilterImpl openIDConnectClaimFilter) {
+    public static void setOpenIDConnectClaimFilter(OpenIDConnectClaimFilterImpl openIDConnectClaimFilter) {
```

Comment:
Why do we need this to be static?

Reply:
Due to https://github.com/wso2/product-is/issues/11143